base/pom.xml

```diff
@@ -7,12 +7,16 @@
 	<parent>
 		<groupId>edu.iu.util</groupId>
 		<artifactId>iu-java-parent</artifactId>
-		<version>7.0.1</version>
+		<version>7.0.2-SNAPSHOT</version>
 	</parent>
 
 	<artifactId>iu-java-base</artifactId>
 	<name>IU Base Utilities Module</name>
 
+	<properties>
+		<argLine>-ea -Xmx1g --add-modules ALL-MODULE-PATH -XX:+EnableDynamicAgentLoading</argLine>

```

Comment:
@mwfyffeiu  You may have explained this before but forget why we need to EnableDynamicAgentLoading. Java future release will disallow this feature as a default https://openjdk.org/jeps/451.